### ISSUE TITLE
[A11y] Collection Landing - Headings

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1818]
### Operating System

Windows

### Browser

Edge

### Affected page

Collection Show

### Admin user?

Yes

### Issue Description

The first H1 (`Oregon Digital (Development)`) should probably not be an H1 since it describes the site and not the page.  The second H1 is correct.

The level 3 headings are weird.  They seem not to need to be headings at all, and the second one, "AND have these attributes" only makes sense in context of the first level 3 heading, which is structurally wrong - headings shouldn't depend on siblings for context.

The level 3 headings for facets have the same problem as on the advanced search: they are currently siblings of the "AND have these attributes" heading, which doesn't make sense structurally.

### Suggested Fix

Remove the H1 around Oregon Digital, and turn the "main" (non-facet) H3s into just regular divs or else make sure they "read" nicely within an outline view.

If the facet headings really need to be headings, they should be at a level that makes sense within the confines of the other headings.

**Comment by jechols** [comment]
This has the same question as the advanced search: do we want a fieldset or headings?

**Comment by jechols** [comment]
Got a response that I like from the list regarding the best way to handle groups of filters.  The general answer boils down to "use a fieldset if you can", but if we can't, the "group" role seems perfect.

> I use grouping elements (<fieldset> or <div role="group">) to group elements with a common group label, including filters.
> 
> I find this to be the best user experience, since the group label is programmatically available, the screen reader can treat it with maximum efficiency, and we're not forcing the common label on every single individual control label (either via label or description).
>
> For filters that instantly affect the search results I use toggle buttons (button elements with with an aria-pressed attribute) instead of checkboxes.
>
> The difference is minor but a checkbox is primarily for forms and the expectations is that you need to submit a form (activate a button) for the checkbox state change to impact the webpage.
> Here is how I would code a search result filter with 45 days being active

I'm reformatting Birkir's suggestion for easier display here, but note that this is just one approach and in this example it's for buttons, but we're still using checkboxes since ours aren't immediately activated.

```html
<div role="group" aria-labelledby="gt">
  <h3 id="gt">Include transactions within</h3>
  <button aria-pressed="false">Last 30 days</button>
  <button aria-pressed="true">Last 45 days</button>
  <button aria-pressed="false">Last 6 months</button>
</div>
```
